submission/submission-handlers/src/test/kotlin/ac/uk/ebi/biostd/handlers/listeners/SubmissionNotificationsListenerTest.kt

```diff
@@ -79,7 +104,7 @@ class SubmissionNotificationsListenerTest(
 
         verify(exactly = 0) {
             webConsumer.getExtSubmission("ext-tab-url")
-            rtNotificationService.notifySuccessfulSubmission(submission, "Dr Owner", "ui-url")
+            rtNotificationService.notifySuccessfulSubmission(submission, "Dr Owner", "ui-url", "st-url")
```

Comment:
as mention before I think verify never should always has specific value. problem is that imagine this is call with "st-url-2" assertion will not fail so test will pass.  I think is simpler and better for test to use any if zero interactiosn scenarios. Also I think is simple to reader to acknowledge this method should not be called rather than this method should not be called with X and Y and Z parameters.